app/controllers/api/v1/answers_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  module V1
+    module Question
+    class AnswersController < BaseController
+      def index
+        binding.pry
+        @answer = Question.find(params[:id]).answers
```

Comment:
- Why do you use an instance variable `@answer`
- better use `.find_by' than `.find`